### DISCUSSION TITLE
[4.0] Codemirror plugin settings (preview)

### DISCUSSION THREAD

**Discussion by C-Lodder** [joomla/joomla-cms#28211] [2. Cold Features]
### Steps to reproduce the issue

1. Go to your Plugin Manager and open the Codemirror plugin
2. Navigate to the "Appearance Options" tab
3. Select any theme from the dropdown

### Expected result

Live preview

### Actual result

Required to click "Save" to view the preview.

![screeny](https://user-images.githubusercontent.com/2019801/75822110-4eb2b180-5d97-11ea-9b3e-1c3fa560e191.png)

**Comment by brianteeman** [comment]
Probably related to this js error

![image](https://user-images.githubusercontent.com/1296369/75824803-15307500-5d9c-11ea-8112-fbe67de40770.png)
